pygem/nurbshandler.py

```diff
@@ -205,20 +222,386 @@ def write(self, mesh_points, filename, tolerance=None):
 			wire = wire_maker.Wire()
 
 			# trimming the surfaces
-			brep_surf = BRepBuilderAPI_MakeFace(occ_face.GetHandle(), wire).Shape()
+			brep_surf = BRepBuilderAPI_MakeFace(occ_face.GetHandle(),
+												wire).Shape()
 			compound_builder.Add(compound, brep_surf)
 			n_faces += 1
 			faces_explorer.Next()
 		self.write_shape_to_file(compound, self.outfile)
 
+
+	def parse_face(self, topo_face):
+		"""
+		Method to parse a single Face (a single patch nurbs surface).
+		It returns a matrix with all the coordinates of control points of the
+		Face and a second list with all the control points related to the
+		Edges of the Face.
+
+		:param topo_face: the input Face
+
+		:return: mesh_points_face: it is a `n_points`-by-3 matrix containing the
```

Comment:
You can merge these two returns by creating a single variable that will be returned. Please check also the other methods.